app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/activity_movie_details"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:background="@color/background_movie_details">
+
+    <ImageView
+        android:id="@+id/act_movie_details_btn_close"
+        android:layout_width="?attr/actionBarSize"
+        android:layout_height="?attr/actionBarSize"
+        android:padding="@dimen/spacing_small"
+        android:src="@drawable/ic_close_white_48dp"
+        android:background="@color/background_btn_close"/>
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical"
+                android:paddingLeft="@dimen/spacing_small"
+                android:paddingRight="@dimen/spacing_small">
+
+                <TextView
+                    android:id="@+id/act_movie_details_tv_title"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center_horizontal"
+                    android:gravity="center"
+                    android:text="Star Wars Collection"
+                    android:textSize="@dimen/font_large"
+                    android:textStyle="bold"
+                    android:textColor="@color/text_movie_title"/>
+
+                <TextView
+                    android:id="@+id/act_movie_details_tv_release_date"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="end"
+                    android:text="20/10/1892"
+                    android:textSize="@dimen/font_tiny"
+                    android:textColor="@color/text_movie_title"/>
```

Comment:
You mean the textColor? I used text_movie_title because both texts use the same color, but I understand that this can be a little confusing and that I should had created a different color resource.